apps/api/src/common/slug-generator.ts

```diff
@@ -1,7 +1,10 @@
 import { PrismaService } from '@/prisma/prisma.service'
 import { Workspace } from '@prisma/client'
 
-const incrementSlugSuffix = (foundSlug: string, baseSlug: string): string => {
+export const incrementSlugSuffix = (
+  foundSlug: string,
```

Comment:
Maybe rename this to `existingSlug`

Reply:
This function's is to get incremented value of the slug. That's why I kept the name that way

Reply:
Yeah just rename that.